app/locations/controllers.js

```diff
@@ -24,6 +24,21 @@ async function locations(req, res, next) {
     req.session.user.locations = userLocations
   }
 
+  if (userPermissions.includes('contract_delivery_manager')) {
```

Comment:
If the user has a supplier ID and the `contract_delivery_manager` does that make the call above uneeded?

Reply:
They won't have supplier Ids, those come from the group (so Serco PECS or Geoamey PECS), and CDMs won't have that group. I don't know if it's worth complicating the per supplier call with a check for CDMs, there are very few CDMs and the chance of them being accidentally added to a supplier group is even lower.